### PR TITLE
[TypeDeclaration] Fix TypedPropertyFromStrictGetterMethodReturnTypeRector for default type

### PR DESCRIPTION
**Pull request by TomasVotruba**

### CODE REVIEW

File: rules/TypeDeclaration/Rector/Property/TypedPropertyFromStrictGetterMethodReturnTypeRector.php [hunk 1]
```diff
@@ -74,54 +71,78 @@ public function getName(): string|null
...
+                continue;
+            }
+
+            if (! $this->parentPropertyLookupGuard->isLegal($property)) {
+                continue;
+            }
+
+            $getterReturnType = $this->getterTypeDeclarationPropertyTypeInferer->inferProperty($property);
+            if (! $getterReturnType instanceof Type) {
+                return null;
```
Comment (samsonasik) [inline rules/TypeDeclaration/Rector/Property/TypedPropertyFromStrictGetterMethodReturnTypeRector.php]: should be continue to next property

File: rules/TypeDeclaration/Rector/Property/TypedPropertyFromStrictGetterMethodReturnTypeRector.php [hunk 2]
```diff
@@ -74,54 +71,78 @@ public function getName(): string|null
...
+            }
+
+            if ($getterReturnType instanceof MixedType) {
+                return null;
```
Comment (samsonasik) [inline rules/TypeDeclaration/Rector/Property/TypedPropertyFromStrictGetterMethodReturnTypeRector.php]: should be continue to next property

File: rules/TypeDeclaration/Rector/Property/TypedPropertyFromStrictGetterMethodReturnTypeRector.php (hunk 1 above)
Comment (TomasVotruba) [reply rules/TypeDeclaration/Rector/Property/TypedPropertyFromStrictGetterMethodReturnTypeRector.php]: Done :+1:

File: rules/TypeDeclaration/Rector/Property/TypedPropertyFromStrictGetterMethodReturnTypeRector.php (hunk 2 above)
Comment (TomasVotruba) [reply rules/TypeDeclaration/Rector/Property/TypedPropertyFromStrictGetterMethodReturnTypeRector.php]: Done :+1:

**Pull request merged**